spec/overcommit/hook/pre_commit/rake_target_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+describe Overcommit::Hook::PreCommit::RakeTarget do
+  let(:config)  { Overcommit::ConfigurationLoader.default_configuration }
+  let(:context) { double('context') }
+  subject { described_class.new(config, context) }
+
+  # context 'without targets parameters' do
+  #   let(:result) { double('result') }
+  #   it 'raises' do
+  #     expect { subject }.to raise_error(RuntimeError)
```

Comment:
this fails for me (added a message checkt too):

```
    
    Failure/Error: expect { subject }.to raise_error(RuntimeError,
      /RakeTarget: targets parameter is empty.*/)
      expected RuntimeError with message matching /RakeTarget:
      targets parameter is empty.*/ but nothing was raised
```

(f4a84b4b4c993ed32c5fdfe5e964283707c23dda)